app/src/main/java/org/metabrainz/mobile/fragment/ArtistBioFragment.java

```diff
@@ -58,12 +58,50 @@ public void onActivityCreated(Bundle savedInstanceState) {
     }
 
     private void findViews(View layout) {
-        bioPicture = layout.findViewById(R.id.bio_picture);
-        yearsActive = layout.findViewById(R.id.years_active);
-        bioText = layout.findViewById(R.id.bio_text);
-    }
-
-    @Override
+        artistType = layout.findViewById(R.id.artist_type);
+        artistGender = layout.findViewById(R.id.artist_gender);
+        artistArea = layout.findViewById(R.id.artist_area);
+        artistLifeSpan = layout.findViewById(R.id.life_span);
+    }
+
+    private void getArtistWiki(){
+        String title = "";
+        for(Link link: artist.getRelations()){
+            if(link.getType().equals("wikipedia")) {
+                title = link.getPageTitle();
+                break;
+            }
+            if (link.getType().equals("wikidata")){
+                title = link.getPageTitle();
+                //TODO:Get Wiki url from wiki data id
+                break;
+            }
+        }
+        if (!title.isEmpty())
+            artistViewModel.getArtistWiki(title).observe(this,
+                wiki -> wikiTextView.setText(wiki.getExtract()));
```

Comment:
⚠️ Be careful, if wiki is null the app crashes because will try to `.getExtract()` from null.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.metabrainz.mobile.api.data.ArtistWikiSummary.getExtract()' on a null object reference
        at org.metabrainz.mobile.fragment.ArtistBioFragment.lambda$getArtistWiki$0(ArtistBioFragment.java:82)
```

Reply:
Yes, this was a quick one to check if the api responds well. Will add counter checks and error messages.